examples/nextjs-clerk-hacker-news/pages/index.tsx

```diff
@@ -0,0 +1,152 @@
+import { gql, useQuery } from '@apollo/client'
+import Head from 'components/head'
+import ItemList from 'components/item-list'
+import { ItemsListQuery } from 'gql/graphql'
+import Link from 'next/link'
+import { useAuth } from '@clerk/nextjs'
+import { graphQlRequestClient } from 'lib/request'
+
+const ITEMS_LIST_QUERY = gql`
+  query ItemsList($after: String) {
+    itemCollection(first: 4, after: $after) {
+      pageInfo {
+        endCursor
+        hasNextPage
+      }
+      edges {
+        node {
+          id
+          title
+          comments(first: 100) {
+            edges {
+              __typename
+            }
+          }
+          votes(first: 100) {
+            edges {
+              node {
+                id
+                positive
+                user {
+                  id
+                }
+              }
+            }
+          }
+          author {
+            id
+            name
+            imageUrl
+          }
+          url
+          createdAt
+        }
+      }
+    }
+  }
+`
+
+const Home = (props: { data: ItemsListQuery }) => {
+  const { isSignedIn } = useAuth()
+  const {
+    data: clientData,
+    loading,
+    error,
+    fetchMore
+  } = useQuery<ItemsListQuery>(ITEMS_LIST_QUERY, {
+    skip: !isSignedIn,
+    notifyOnNetworkStatusChange: true
+  })
+
+  const data = clientData ?? props.data
+
+  return (
+    <>
+      <Head>
+        <title>Grafnews | Feed</title>
+      </Head>
+      <div className="space-y-8">
+        <div className="bg-indigo-600 p-4 border border-b-4 border-indigo-300 space-y-4 sm:space-y-0 sm:flex items-center justify-between sm:space-x-4">
+          <h2 className="text-white text-2xl ">
+            Grafbase is open to everyone, start building your frontend with the
+            next-gen GraphQL platform!
+          </h2>
+          <div>
+            <a
+              href="https://grafbase.com/register"
```

Comment:
Outdated: path no longer exists 
Also, I would add target blank

Reply:
It does exist, its a path to our website: https://grafbase.com/register. 

Yes, sure!

Reply:
Maybe we add a redirect on our website? It seems right now we show the same page at `/sign-up` and `/register`?

Reply:
It should indeed be a permanent redirect

Reply:
It's a rewrite right now in next.config.

<img width="436" alt="image" src="https://user-images.githubusercontent.com/27777508/201648553-18cf697f-687a-4a77-8757-4ac12c897556.png">


I don't see anything wrong with this, but could be a redirect, yes.

Reply:
It should be a 301 redirect.

Reply:
Sorry, when I opened the link I saw 404 that why I mentioned it